install/kubernetes/helm/istio-remote/values.yaml

```diff
@@ -48,8 +48,8 @@ global:
   rbacEnabled: true
 
   ## imagePullSecrets for all ServiceAccount. Must be set for any clustser configured with privte docker registry.
-  # imagePullSecrets:
-  #   - name: "private-registry-key"
+  imagePullSecrets:
```

Comment:
@morvencao Can you please add some comments here for `imagePullSecrets`, it can be used by Pod or ServiceAccount?

Reply:
@costinm @gyliu513 
We firstly create secret of docker registry and then add it to `imagePullSecrets` of serviceAccount, after that any new created `pods` with the `serviceAccount` will have automatically this imagePullSecrets added to their spec as follow:
```
spec:
  imagePullSecrets:
  - name: myregistrykey
```
So that we don't need to add the secret to each pod manually.

Please refer to: https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/#add-imagepullsecrets-to-a-service-account

Reply:
@morvencao How about add some comments to clarify this?

Such as 
```
`imagePullSecrets` can not only be used in Pod, but also ServiceAccount. Refer to https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/#add-imagepullsecrets-to-a-service-account for detail.
```

Reply:
Yes,  `imagePullSecrets` can not only be used in Pod, but also ServiceAccount. 
For each `serviceaccount`, we add `imagePullSecrets` to its spec, so that every pod binding with the `serviceaccount` will have same `imagePullSecrets ` to use for pulling any images.